src/org/zaproxy/zap/control/AddOn.java

```diff
@@ -54,6 +57,14 @@
 import org.zaproxy.zap.extension.pscan.PluginPassiveScanner;
 
 public class AddOn  {
+
+	/**
+	 * The name of the manifest file, contained in the add-ons.
+	 * 
+	 * @since TODO add version
+	 */
+	public static final String MANIFEST_FILE_NAME = "ZapAddOn.xml";
+
 	public enum Status {unknown, example, alpha, beta, weekly, release}
 	
 	private static ZapRelease v2_4 = new ZapRelease("2.4.0");

```

Comment:
No, that version was created to prevent loading (older) add-ons that could be incompatible with newer ZAP version (2.4.0).